docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+services:
+  cinema_service:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8001:8000"
+    volumes:
+      - ./:/app
+      - my_meda:/files/media
```

Comment:
There is a typo in the volume name `my_meda`. It should probably be `my_media` to match the intended directory for media files. Ensure consistency in naming to avoid issues with volume mounting.